tests/unit/info-keysizes.tcl

```diff
@@ -471,6 +475,16 @@ proc test_all_keysizes { {replMode 0} } {
         run_cmd_verify_hist {} {__EVAL_DB_HIST__ 0} 1
     } {} {cluster:skip}
     
+    test "KEYSIZES - Test DEBUG KEYSIZES-HIST-ASSERT command" {
+        # Test based on debug command rather than __EVAL_DB_HIST__
+        start_server {} {
+            r DEBUG KEYSIZES-HIST-ASSERT 1
+            r FLUSHALL
+            createComplexDataset r 100
+            createComplexDataset r 100 {useexpire usehexpire}            
```

Comment:
```suggestion
            createComplexDataset r 100 {useexpire usehexpire}
            r DEBUG KEYSIZES-HIST-ASSERT 0
```

Reply:
@moticless you missed `r DEBUG KEYSIZES-HIST-ASSERT 0`, otherwise, this flag will affect other tests when running external test.